crud/common/const.lua

```diff
@@ -0,0 +1,6 @@
+local const = {}
+
+const.RELOAD_RETRIES_NUM = 1
+const.RELOAD_SCHEMA_TIMEOUT = 30 -- half of minute
```

Comment:
Even with 1k rps you will create 30k fibers. It's too huge amount. I think you need to decrease to 3-5 seconds timeout.